.eslintrc.js

```diff
@@ -100,6 +100,12 @@ module.exports = {
 		'jsx-a11y/anchor-is-valid': 0,
 	},
 	overrides: [
+		{
+			files: [ '_inc/client/**/*' ],
+			globals: {
+				__AppFeatures__: 'readonly',
```

Comment:
Is this "globals" as in `window` or globals for the app bundle? Should we prefix this with `Jetpack`